cmd/kube-controller-manager/app/options/persistentvolumebindercontroller.go

```diff
@@ -63,6 +69,17 @@ func (o *PersistentVolumeBinderControllerOptions) Validate() []error {
 		return nil
 	}
 
+	cleanedList := strings.TrimSpace(o.DialHostCIDRDenylist)
+	if cleanedList == "" {
+		return nil
+	}
+
 	errs := []error{}
+	for _, cidr := range strings.Split(cleanedList, ",") {
+		if _, _, err := net.ParseCIDR(strings.TrimSpace(cidr)); err != nil {
```

Comment:
Can we validate this at a higher layer? especially if we need to support multiple volume controllers?

Reply:
The string is already validated at cmd/kube-controller-manager/app/options/persistentvolumebindercontroller.go:67, is that what you were thinking?